src/filters/PoseEstimatorLinear.h

```diff
@@ -8,6 +8,8 @@
 namespace filters {
 
 /**
+ * @brief Uses a Kalman Filter to estimate the robot pose.
```

Comment:
Admittedly, I never looked into this before, but what is the difference between `PoseEstimator` and `PoseEstimatorLinear`? All I know is that `PoseEstimatorLinear` takes a really long time to compile

Reply:
`PoseEstimatorLinear` uses a vanilla KF while `PoseEstimator` uses an EKF. I wasn't very imaginative with the naming, so it might not be very clear lol.

I'm not sure why it would take forever to compile? Could be because of lots of templating, maybe?